pages/dkp/kaptain/2.0.0/configuration/notebook-servers-configuration/index.md

```diff
@@ -22,54 +22,49 @@ This allows notebook-specific workloads to run on specific nodes from a pool of
 ```yaml
-notebookTolerationGroups:
-  - groupKey: "notebooks"
-    displayName: "Notebooks Node Group"
-    tolerations:
-      - key: "dedicated"
-        operator: "Equal"
-        value: "notebook"
-        effect: "NoExecute"
-notebookAffinityConfig:
-  - configKey: "notebook-affinity-config"
-    displayName: "Notebook Affinity Configuration"
-    affinity:
-      nodeAffinity:
-        requiredDuringSchedulingIgnoredDuringExecution:
-          nodeSelectorTerms:
-          - matchExpressions:
-            - key: topology.kubernetes.io/region
-              operator: In
-              values:
-              - us-west-1
-              - us-west-2
-        preferredDuringSchedulingIgnoredDuringExecution:
-        - weight: 1
-          preference:
-            matchExpressions:
-            - key: another-node-label-key
-              operator: In
-              values:
-              - another-node-label-value
+core:
+  notebook:
+    notebookTolerationGroups:
+      - groupKey: "notebooks"
+        displayName: "Notebooks Node Group"
+        tolerations:
+          - key: "dedicated"
+            operator: "Equal"
+            value: "notebook"
+            effect: "NoExecute"
+    notebookAffinityConfig:
+      - configKey: "notebook-affinity-config"
+        displayName: "Notebook Affinity Configuration"
+        affinity:
+          nodeAffinity:
+            requiredDuringSchedulingIgnoredDuringExecution:
+              nodeSelectorTerms:
+              - matchExpressions:
+                - key: topology.kubernetes.io/region
+                  operator: In
+                  values:
+                  - us-west-1
+                  - us-west-2
+            preferredDuringSchedulingIgnoredDuringExecution:
+            - weight: 1
+              preference:
+                matchExpressions:
+                - key: another-node-label-key
+                  operator: In
+                  values:
+                  - another-node-label-value
 ```
 
 You can set any other desired operator parameters in this file as well, so that you have a single file with all the operator configurations.
 Please refer to the [Toleration v1 core](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#toleration-v1-core)
 and [Affinity v1 core](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#affinity-v1-core)
 pages in the Kubernetes API documentation to see all the supported fields.
 
-Specify the parameters file as an argument to the `kubectl kudo install` command during the installation:
-```bash
-kubectl kudo install \
-    --instance kaptain \
-    --namespace kubeflow \
-    --create-namespace ./kubeflow-1.4.0_1.3.0.tgz \
-    --parameter-file parameters.yaml
-```
+Install Kaptain by following the [Deploy Kaptian][deploy-kaptain] documentation. In case of update, edit the `ConfigMap` used for Kaptain installation (or the default one, created by Flux controller). 
```

Comment:
```suggestion
Install Kaptain by following the [Deploy Kaptain][deploy-kaptain] documentation. In case of update, edit the `ConfigMap` used for Kaptain installation (or the default one, created by Flux controller). 
```